cmake/Modules/VolkPython.cmake

```diff
@@ -100,6 +100,11 @@ endmacro(VOLK_PYTHON_CHECK_MODULE)
 # Sets the python installation directory VOLK_PYTHON_DIR
 ########################################################################
 if(NOT DEFINED VOLK_PYTHON_DIR)
+VOLK_PYTHON_CHECK_MODULE("distutils" "distutils" "True" HAVE_DISTUTILS)
+if(NOT HAVE_DISTUTILS)
+    message(FATAL_ERROR "Debian does not include the 'distutils' module from Python’s Standard Library in its Python package; install 'python-disutils' or 'python3-distutils'")
```

Comment:
Minor comment: I am a bit on the edge with the wording. This is also true for ubuntu and probably also their derivatives.

Maybe reword to start focus on the module missing and add the help text afterward?: 
``` 
Python module 'distutils' from Python’s Standard Library not found. On Debian install 'python3-distutils'.
```